src/tarballs/build.ts

```diff
@@ -22,8 +22,8 @@ const pack = async (from: string, to: string) => {
   await mkdir(path.dirname(to), {recursive: true})
   log(`packing tarball from ${prettifyPaths(path.dirname(from))} to ${prettifyPaths(to)}`)
   to.endsWith('gz')
-    ? await exec(`tar czf ${to} ${path.basename(from)}`, {cwd})
-    : await exec(`tar cfJ ${to} ${path.basename(from)}`, {cwd})
+    ? await exec(`tar czf ${to} ${path.basename(from)}${process.platform === 'win32' ? ' --force-local' : ''}`, {cwd})
```

Comment:
These are necessary so paths behave properly on windows. Similar to  https://github.com/oclif/oclif/blob/458437e5572b496d7eaf84c33b710eaf2f514b33/src/tarballs/build.ts#L201